sagemaker-inference-recommender/auto-scaling/optimize_endpoint_scaling.ipynb

```diff
@@ -0,0 +1,1018 @@
+{
```

Comment:
<p>English -&gt; "To arrive to the correct scaling property" -&gt; "To determine the correct scaling property'</p><p> </p><p>"The Default job, is a recommendation job, however we can limit the search to a single instance by passing the instance type in the supported instances." -&gt; "The default job is a recommendation job. However we can limit the search to a single instance by passing the single instance type to the paramter instance_type"</p><p> </p><p>"The service then provisioning the endpoint, gradually increases the request and stops when the benchmark reaches saturation or if the endpoint invoke API call fails for 1% of the results. " -&gt; "The service then provisions the endpoint. It gradually increases the request and stops when the benchmark reaches saturation or when 1% of API invocations fail. "</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3705/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>